src/java.desktop/share/native/libharfbuzz/hb-icu.h

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
We don't need/use hb-icu.h and hb-icu.cc, so why are they here ?
They were there in harfbuzz source for a long time and we've never needed them before and
given what their purpose is, I don't see why we need them now.
Are you sure you followed the process of adding in ONLY new files which were needed so that we successfully
compiler ?
Please double check ALL new files you added are really needed.

Reply:
@prrace hb-icu.h was referenced in hb-unicode.cc (existing file), but I missed to see that it is included only when certain flags are set. Thank you for catching it.

I will double check all the newly added files again.